packages/core/src/RenderingEngine/StackViewport.ts

```diff
@@ -552,7 +546,7 @@ class StackViewport extends Viewport implements IStackViewport, IImagesLoader {
       mapper.setPreferSizeOverAccuracy(true);
     }
 
-    if (imageData.getPointData().getNumberOfComponents() > 1) {
+    if (imageData.getPointData().getScalars().getNumberOfComponents() > 1) {
```

Comment:
This doesn't agree with the issue text about how to get number of components - should it use the same format as the issue text?